autospec/build.py

```diff
@@ -159,7 +159,10 @@ def failed_pattern(line, pattern, verbose, buildtool=None):
                 org = match.group(1)
                 name = match.group(2)
                 ver = match.group(3).replace('-', '.')
-                mvn_provide = f'mvn({org}:{name}:jar) = {ver}'
+                if "-parent" in name or "-pom" in name or "-bom" in name:
```

Comment:
Are these substrings suffixes? If so, I suggest to do:

```
if re.search("-(parent|pom|bom)$", name):
    ...
```

Reply:
Good suggestion ;)